languages/csharp/exercises/concept/exceptions/.meta/Example.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+static class SimpleCalculator
+{
+    public static string ErrorLog {get; set;}
+    public static int Calculate(int operand1, int operand2, string operation)
+    {
+        int result = 0;
+        ErrorLog = "";
+        try
+        {
+            switch(operation)
+            {
+                case "+":
+                    result = Addition(operand1, operand2);
+                    break;
+                case "*":
+                    result = Multiplication(operand1, operand2);
+                    break;
+                case "/":
+                    result = Division(operand1, operand2);
+                    break;
+                default:
+                    throw new InvalidOperationException($"Operation {operation} does not exist");
```

Comment:
I think the appropriate exception here is a [`ArgumentOutOfRangeException`](https://docs.microsoft.com/en-us/dotnet/api/system.argumentoutofrangeexception?view=netcore-3.1).

```suggestion
                    throw new ArgumentOutOfRangeException($"Operation {operation} does not exist");
```